tinygrad/runtime/ops_gpu.py

```diff
@@ -43,12 +43,13 @@ def __init__(self, size, dtype, device='0'):
 
   def _copyin(self, x:np.ndarray):
     assert not self.dtype.name.startswith("image"), f"can't copyin images {self.dtype}"
-    self.event = cl.enqueue_copy(CL.cl_queue[self._buf.device], self._buf, np.require(x, requirements='C'), is_blocking=False)
+    self.event = cl.enqueue_copy(CL.cl_queue[self._buf.device], self._buf, np.require(x, requirements='C'), is_blocking=True)
```

Comment:
Why does this have to change to blocking?

Reply:
was testing stuff, forgot to revert